front/lib/api/wrappers.ts

```diff
@@ -134,7 +134,12 @@ export function withSessionAuthenticationForWorkspace<T>(
 
 /**
  * This function is a wrapper for Public API routes that require authentication for a workspace.
- * It must be used on all routes that require workspace authentication (prefix: /v1/w/[wId/]).
+ * It must be used on all routes that require workspace authentication (prefix: /v1]).
+ *
+ * opts.isWorkspaceRoute is used to determine if the route is under the /v1/w/[wId/] prefix.
+ * We have one route (/v1/me) that is not a workspace route.
+ *
+ * opts.forceAuthWithToken is used to determine if the route requires an auth token for authentication.
```

Comment:
What do we plan to use this one for? Limiting some endpoints of our public API to Auth0 token?